fili-core/src/main/java/com/yahoo/bard/webservice/async/jobs/stores/ApiJobStore.java

```diff
@@ -51,4 +53,21 @@
      * @return An Observable that emits a stream of all the JobRows in the store
      */
     Observable<JobRow> getAllRows();
+
+    /**
+     * This method takes a Set of ApiJobStoreFilters, ANDS them by default, and returns a cold observable that emits a
+     * stream of JobRows which satisfy the given filter.
+     * <p>
+     * Every field may not be filterable for every implementation of the `ApiJobStore` as the efficiency of filtering is
```

Comment:
Should replace the markdown style backticks with the JavaDoc `{@code ...}`. I too wish that JavaDocs supported Markdown, but alas this is not Elixir (also I think Java predates Markdown).